testcase/resolve-lock/resolve_lock.go

```diff
@@ -568,7 +568,7 @@ func (c *resolveLockClient) getTs(ctx context.Context) (uint64, error) {
 			ts := oracle.ComposeTS(physical, logical)
 			return ts, nil
 
-		case io.EOF:
+		case io.EOF, context.Canceled:
```

Comment:
Should check which context produces this error, i.e., if `ctx.Err() != nil`, we needn't backoff and retry here and should `return 0, nil` immediately.